src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -483,13 +483,25 @@ public String getServicePhraseName(Interface service) {
   }
 
   /**
+   * The type name of the Grpc server class. This needs to match what Grpc generates for the
+   * particular language.
+   */
+  public String getGrpcServerTypeName(Interface service) {
```

Comment:
Oh yes, getGrpcClient/ServerTypeName sounds language specific to me so I think in the case we probably should change both.